Dockerfile

```diff
@@ -83,11 +79,9 @@ COPY --from=builder /jellyfin /jellyfin
 COPY --from=web-builder /dist /jellyfin/jellyfin-web
 
 EXPOSE 8096
-VOLUME /cache /config
+VOLUME ${JELLYFIN_DATA_DIR} ${JELLYFIN_CACHE_DIR}
```

Comment:
Does this work? I always thought only ARG works outside of RUN statements

Reply:
![image](https://github.com/jellyfin/jellyfin/assets/68083474/5264fb71-ef82-4181-bc1e-bb27e7d8e5ee)
Does this respond to your question?

Reply:
Just so you can see when building image, in exemple it's the arm one 
![image](https://github.com/jellyfin/jellyfin/assets/68083474/9a9db11c-478d-44b0-9670-8e3cd98c114e)